uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/models/instances/leave-request-instance.js

```diff
@@ -125,7 +125,8 @@ define([
             request_type: 'leave',
             uploader: FileUpload.uploader({
               entityTable: 'civicrm_hrleaveandabsences_leave_request',
-              crmAttachmentToken: sharedSettings.attachmentToken
+              crmAttachmentToken: sharedSettings.attachmentToken,
```

Comment:
The following comment is unrelated to this line, but I can't comment on lines that are not marked as diff

When the instance uploads the attachments via `this.uploader`, shouldn't it store them afterwards in the `this.files` property? By looking at
```js
function uploadAttachments() {
  if (this.uploader.queue && this.uploader.queue.length > 0) {
    return this.uploader.uploadAll({ entityID: this.id });
  } else {
    return $q.resolve([]);
  }
}
```
once the files are uploaded nothing happens, the instance is completely unaware of the files that are now associated with it